app/helpers/organisations_helper.rb

```diff
@@ -0,0 +1,11 @@
+module OrganisationsHelper
+
+  def has_access(country)
+    yes = content_tag(:span, '', class: 'fa fa-check') +
+      content_tag(:span, 'Yes')
+    no = content_tag(:span, '', class: 'fa fa-times') +
+      content_tag(:span, 'No')
+    return no unless @adapter
+    @adapter.countries_with_access_ids.include?(country.id) ? yes : no
```

Comment:
this check: `countries_with_access_ids.include?(country.id)` also appears in two other places - suggest making it a method of the adapter.